Add Dinum blogpost (#2965)
matrix-org/matrix.org: content/blog/2025/10/2025-10-21-dinum.md

+++
date = "2025-10-21T12:30:00Z"
title = "Matrix for Public Sector: DINUM joins the Foundation as we launch a new working group!"

[taxonomies]
author = ["Amandine Le Pape"]
category = ["Foundation","Conference","EU"]

[extra]
image = "https://matrix.org/blog/img/DINUM_blog_social-white.png"
+++

## DINUM becoming the first government to join the Matrix.org Foundation

Today [DINUM, the French Interministerial Digital Directorate, has officially announced that they were joining the Matrix.org Foundation](https://www.numerique.gouv.fr/sinformer/espace-presse/dinum-soutien-fondation-matrixorg/) as a Silver member and becomes the first government to join the Matrix.org Foundation!

This is particularly exciting news as it will hopefully set an example for other public sector organisations using Matrix to communicate, and there are many of them.

<!-- more -->

> In France, Matrix is the basis for Tchap, the secure messaging system of choice for public officials. It was selected for its security, but also openness, transparency and sovereignty as it limits dependence on proprietary solutions from outside Europe.

Reviewer: Perhaps
```suggestion
In France, Matrix is the basis for [Tchap](https://tchap.numerique.gouv.fr/), the secure messaging system of choice for public officials. It was selected for its security, but also openness, transparency and sovereignty as it limits dependence on proprietary solutions from outside Europe.
```

## Matrix for Public Sector

This announcement comes a few days after the second edition of the [Matrix Conference](https://2025.matrix.org/). The Conference was held in Strasbourg (France), gathering about 300 attendees, amongst which more than a dozen public sector organisations, brought together by their use of Matrix as a sovereign, secure and resilient communication protocol.

The Conference saw two full days of [public sector track](https://2025.matrix.org/schedule/), shedding light on the various exciting deployments of Matrix by these organisations. All the talks are already available for watching [here](https://www.youtube.com/playlist?list=PLl5dnxRMP1hUgnYEbpEsEEhIqY_KlO3NG), and will be soon published on the website, but you can find a list of the public sector related ones at the bottom of this post.

> We also took the opportunity of having many representatives of these public organisations present in person at the Conference to **officially launch a Matrix for Public Sector working group**. This is the very first working group of its kind that the Foundation sets up and already many other industries are asking for something similar.

Reviewer: Is this supposed to mean a WG in the sense of https://matrix.org/foundation/working-groups/?

Reviewer: ```suggestion
We also took the opportunity of having many representatives of these public organisations present in person at the Conference to **officially launch a Matrix for Public Sector forum**. This is the very first group of its kind that the Foundation sets up and already many other industries are asking for something similar. The group will be proposed to the Governing Board for approval to potentially become an official [Working Group](https://matrix.org/foundation/working-groups/).
```

> We also took the opportunity of having many representatives of these public organisations present in person at the Conference to **officially launch a Matrix for Public Sector forum**. This is the very first group of its kind that the Foundation sets up and already many other industries are asking for something similar. The group will be proposed to the Governing Board for approval to potentially become an official [Working Group](https://matrix.org/foundation/working-groups/).

Reviewer: ```suggestion
We also took the opportunity of having many representatives of these public organisations present in person at the Conference to **officially launch a Matrix for Public Sector forum**. This is the very first group of its kind that the Foundation sets up and already many other industries are asking for something similar. The group will be proposed to the Governing Board for approval to potentially become an official [Working Group](@/foundation/working-groups/index.md).
```
